src/main/scala/de/heikoseeberger/reactiveflows/MessageEventPublisher.scala

```diff
@@ -24,6 +24,7 @@ object MessageEventPublisher {
   def props(mediator: ActorRef, bufferSize: Int) = Props(new MessageEventPublisher(mediator, bufferSize))
 }
 
+// TODO: collapse with FlowEventPublisher
```

Comment:
“Collapsing” means introducing a type parameter for the precise kind of stream element (plus implicit marshaller) and passing those things which differ into the constructor.